noetic/distribution.yaml

```diff
@@ -4373,6 +4373,22 @@ repositories:
       url: https://github.com/MITRE/kvh_geo_fog_3d.git
       version: noetic-devel
     status: maintained
+  l3cam_ros:
+    doc:
+      type: git
+      url: https://github.com/beamaginelidar/l3cam_ros.git
```

Comment:
There seem to be a number of problems in the source repository:

1.  The repository is missing a LICENSE file, which should be a 3-clause BSD (from the package.xml)
2.  While it is not strictly required, it is also recommended to have a license header at the top of every source file.
3.  It would be good to put something in the README.md that describes what this package is meant to do.  That would help us figure out whether it is appropriate for the rosindex and whether the name is appropriate.
4.  There seems to be a binary archive checked into the source repository, which probably should not be there: https://github.com/beamaginelidar/l3cam_ros/blob/master/include/libL3cam/libL3Cam.a

Reply:
Working on these.
The binary archive is the SDK the node depends on, which we first thought it to be integrated. Is it good measure having it integrated or should we externalize it and specify the dependency?
Thank you

Reply:
> The binary archive is the SDK the node depends on, which we first thought it to be integrated. Is it good measure having it integrated or should we externalize it and specify the dependency?

You should externalize it as much as possible.  That way the dependency is clear.